### PR TITLE
WRO-2902: Fixed `enact serve` fails with multiple ports

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
With CLI 5.0.0-alpha.1, when some sample is running on 8080 port, `enact serve` is failed on another port saying
> Would you like to run the app on another port instead? … yes
 Error: listen EADDRINUSE: address already in use 0.0.0.0:8080
 at Server.setupListenHandle [as _listen2] (net.js:1331:16)
 at listenInCluster (net.js:1379:12)
 at doListen (net.js:1516:7)
 at processTicksAndRejections (internal/process/task_queues.js:83:21)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Move `devServer` options in webpack.config.js to `devServerConfig` in serve.js
- Add a parameter for `devServerConfig` to override resolvePort variable as `port`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
- Restore `devServer` options that Enact CLI used.
https://github.com/enactjs/cli/pull/268/files#diff-c49a22733230fd83e34ff3bc78014ee352ad46ce10c7c502dd2a7c3982adb0a5L321
- Migrated options to current webpack-dev-server version.
https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md
- `stats` in webpack.config.js is set such that only errors or warnings are printed by webpack.
image

### Links
[//]: # (Related issues, references)
WRO-2902

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### pack, serve
 
+* Fixed `enact serve` fails with multiple ports by overriding resolvePort as devServer port.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]: Please move this under ### serve section.

Comment (hong6316) [reply CHANGELOG.md:5]: OK

File: config/webpack.config.js [hunk 2]
```diff
@@ -186,6 +186,8 @@ module.exports = function (env, isomorphic = false, ilibAdditionalResourcesPath)
 	};
 
 	return {
+		// Webpack noise constrained to errors and warnings
+		stats: 'errors-warnings',
```
Comment (MikyungKim) [inline config/webpack.config.js:190]: Let's move this after bail.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ## unreleased
 
-### pack, serve
+### serve
+
+* Fixed `enact serve` fails with multiple ports by overriding resolvePort as devServer port.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Fixed `enact serve` fails with multiple ports.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ## unreleased
 
-### pack, serve
+### serve
+
+* Fixed `enact serve` fails with multiple ports.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Fixed `enact serve` fails to open another port when the default port is busy.
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**